src/cmd/terraform.go

```diff
@@ -310,30 +310,49 @@ func exportServices(c *opslevel.Client, shell *os.File, directory string) {
 func exportTeams(c *opslevel.Client, config *os.File, shell *os.File) {
 	shell.WriteString("# Teams\n")
 
-	teamConfig := `resource "opslevel_team" "%s" {
-  name = "%s"
-  manager_email = "%s"
-  parent = "%s"
-  %s
-  %s
+	teamConfig := `resource "opslevel_team" "%s" {%s
 }
+
 `
 	resp, err := c.ListTeams(nil)
 	teams := resp.Nodes
 	cobra.CheckErr(err)
 	for _, team := range teams {
+		teamBody := ""
```

Comment:
Can we please introduce `strings.Builder` here?
We can set `teamBody := strings.Builder{}` just above this for loop then:
1. sb.WriteString("strings below")
2. sb.String()  // to return the whole string to `config.WriteString(templateConfig()` below
3. sb.Reset() // to reset the underlying buffer at the end of each loop iteration

Running this command is surprisingly slow and this would be a small step in a more optimized direction IMO

Reply:
Added!